src/Directory.Packages.props

```diff
@@ -14,5 +15,6 @@
     <PackageVersion Include="System.Runtime.Caching" Version="8.0.0" />
     <PackageVersion Include="System.Security.Principal.Windows" Version="5.0.0" />
     <PackageVersion Include="System.Text.Json" Version="8.0.5" />
+    <PackageVersion Include="Yarp.ReverseProxy" Version="2.3.0" />
```

Comment:
I know you are not using that here yet, but as FYI Aspire also has a custom integration for Yarp.

Reply:
Yeah, I took a look at it, but it seems to require a setup with a config file which isn't how we've been using it since there's a single fallback. Maybe I missed something, but it seems that in its current state it's not a great fit for how customers set up incremental migration setups.